services/skus/service.go

```diff
@@ -2488,3 +2547,42 @@ func shouldUpdateOrderStripeSubID(ord *model.Order, subID string) bool {
 
 	return false
 }
+
+func newRadomGateway(env string) (radom.Gateway, error) {
+	switch env {
+	case "development", "staging":
+		return radom.Gateway{
+			Managed: radom.Managed{
+				Methods: []radom.Method{
```

Comment:
suggestion: Since the slice is going to escape to the heap anyway (since it's returned as part of a copy), we might as well return just a pointer to `radom.Gateway` and use it as pointer as opposed to allowing copies.